spec/migrations/20190327132620_dialog_field_load_values_on_init_spec.rb

```diff
@@ -0,0 +1,21 @@
+require_migration
+
+describe DialogFieldLoadValuesOnInit do
+  let(:dialog_field_stub) { migration_stub(:DialogField) }
+
+  migration_context :up do
+    before do
+      [true, false, nil].each do |show|
+        [true, false, nil].each do |load|
+          dialog_field_stub.create!(:show_refresh_button => show, :load_values_on_init => load)
+        end
+      end
+
+      migrate
+    end
+
+    it "updates load_values_on_init" do
+      expect(dialog_field_stub.pluck(:load_values_on_init)).to eq([true, false, nil, true, true, true, true, true, true])
```

Comment:
Can you add an explicit sort here so we don't get bit by locale or OS default sorting rules?

Reply:
I was hoping we can reliably get them in order they were created.

Is there any nice way of doing that?
(But I can live with sort too...)

Reply:
Added a `sort_by(:id)` since we probably *can* rely on the ids being ascending, right?

And split into 3 lines to better show the relation.